src/randoop/test/ValidityCheckingVisitor.java

```diff
@@ -3,30 +3,44 @@
 import randoop.ExceptionalExecution;
 import randoop.ExecutionOutcome;
 import randoop.sequence.ExecutableSequence;
+import randoop.sequence.SequenceExceptionError;
 import randoop.test.predicate.ExceptionPredicate;
 
 /**
  * A {@code ValidityCheckingVisitor} checks for occurrences of exceptions that
  * have been tagged as invalid behaviors as represented by a 
  * {@code ExceptionPredicate}.
+ * Also, either ignores or reports flaky test sequences --- an input 
```

Comment:
In ASCII, use "--" to represent an em dash.  (In LaTeX, --- represents an em dash.)